packages/wpcom-template-parts/src/universal-footer-navigation/index.tsx

```diff
@@ -477,7 +464,9 @@ const UniversalNavbarFooter = ( {
 	const isEnglishLocale = useIsEnglishLocale();
 	const pathNameWithoutLocale =
 		currentRoute && removeLocaleFromPathLocaleInFront( currentRoute ).slice( 1 );
-	const [ automatticBranding, setAutomatticBranding ] = useState( { article: '', noun: '' } );
+	const [ automatticBranding, setAutomatticBranding ] = useState< React.ReactChild >(
```

Comment:
The `React.ReactChild` type (or rather `React.JSX.Element`) will be automatically inferred from the default value, doesn't need to be specified.